Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM yastdevel/storage-ng
+RUN zypper --gpg-auto-import-keys --non-interactive in yast2-partitioner
```

Comment:
Alternatively you can add the package to https://github.com/yast/docker-storage-ng/blob/master/Dockerfile, then it will be available for all packages at Travis.